client/src/App.js

```diff
@@ -29,13 +29,15 @@ const App = () => {
     <Provider store={store}>
       <BrowserRouter history={history}>
         <Navbar />
-        <Route exact path='/' component={Landing} />
-        <Route exact path='/checkloggedin' component={CheckLoggedIn} />
-        <SemiPrivateRoute exact path='/helform' component={HelForm} />
-        <PrivateRoute exact path='/Dashboard' component={Dashboard} />
-        <PrivateRoute exact path='/helData' component={HELData} />
-        <PrivateRoute exact path='/create' component={CreateTimeTable} />
-        <PrivateRoute exact path='/aboutUs' component={About} />
+        <Route exact path="/" component={Landing} />
+        <div className="body-wrapper">
+          <Route exact path="/checkloggedin" component={CheckLoggedIn} />
+          <SemiPrivateRoute exact path="/helform" component={HelForm} />
+          <PrivateRoute exact path="/Dashboard" component={Dashboard} />
+          <PrivateRoute exact path="/helData" component={HELData} />
+          <PrivateRoute exact path="/create" component={CreateTimeTable} />
```

Comment:
I would request you to use ESLint and Prettier with the same settnigs as the original project, other wise it will become quite difficult for maintainers to review the PR.

Reply:
There was not ESLint and Prettier config in the repo.

Reply:
Yeah we'll need to add it